lib/runner.js

```diff
@@ -358,11 +363,10 @@ Runner.prototype.hook = function(name, fn) {
   var hooks = suite.getHooks(name);
   var self = this;
 
-  function next(i) {
+  function nextHook(i) {
     var hook = hooks[i];
-    if (!hook) {
-      return fn();
-    }
+    if (!hook) return fn();
```

Comment:
I much prefer the style as-is, but as long as prettier or eslint doesn't complain, I can live with it.